Makefile

```diff
@@ -54,6 +66,35 @@ build-image-ocp-example: build-base-image ## Build a rag-content container image
 build-base-image: ## Build base container image
 	podman build -t $(TORCH_GROUP)-road-core-base -f Containerfile.base --build-arg FLAVOR=$(TORCH_GROUP)
 
+start-postgres:
+	podman run -d --name pgvector --rm -e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
+	 -p $(POSTGRES_PORT):5432 \
+	 -v $(PWD)/postgresql/data:/var/lib/postgresql/data:Z pgvector/pgvector:pg16
+
+start-postgres-debug:
+	mkdir -pv ./postgresql/data
+	podman run --name pgvector --rm -e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
```

Comment:
Should run in detached mode, so that the terminal is free for other tasks.

Reply:
This was intentionally written in that way.  While the `start-postgres` target runs postgres in detatched mode, this `start-postgres-debug` target runs postgres in a terminal showing logs by setting `-c log_statement=all -c log_destination=stderr` options.